packages/parcel-transformer/src/index.ts

```diff
@@ -77,14 +82,16 @@ export default new Transformer<ParcelTransformerOpts>({
     };
   },
 
-  async transform({ asset, config }) {
+  async transform({ asset, config, options }) {
     const ast = await asset.getAST();
     if (!ast) {
       // We will only receive ASTs for assets we're interested in.
       // Since this is undefined (or in node modules) we aren't interested in it.
       return [asset];
     }
 
+    // Disable stylesheet extraction locally due to https://github.com/atlassian-labs/compiled/issues/1306
+    const extract = config.extract && options.mode !== 'development';
```

Comment:
Disable stylesheet extraction on local Parcel. I will update the documentation once this PR is merged.